book/analysis/performance-of-python-types.md

```diff
@@ -5,90 +5,74 @@ position: 4
 layout: default.html
 ---
 
-Now that you have a general idea of big O notation and the differences
-between the different functions, our goal in this section is to tell you
-about the big O performance for the operations on Python lists and
-dictionaries. It is important for you to understand the efficiency of
-these Python data structures because they are the building blocks we
-will use as we implement other data structures in the remainder of the
-book. In this section we will try to give you some intuitive understanding
-of why the performance is what it is, but you will not fully appreciate
-why this is the case until later sections where we dive into the details
-of how lists and dictionaries may be implemented.
+Now that you have a general idea of big O notation and the differences among
```

Comment:
Yeah, I think "idea of big O notation" encapsulates that well enough; good call.